src/main/java/com/climatetree/places/model/EcoName.java

```diff
@@ -16,6 +16,11 @@
 @Table(name="\"ECO_NAME\"")
 public class EcoName implements Serializable {
 
+	/**
+	 * 
+	 */
+	private static final long serialVersionUID = -3223489926850482766L;
```

Comment:
What is this for?

Reply:
this is usually to remove the warning because i am implementing the Serializer class.